source/solvers/heat_transfer.cc

```diff
@@ -972,6 +999,33 @@ HeatTransfer<dim>::read_checkpoint()
     {
       previous_solutions[i] = distributed_previous_solutions[i];
     }
+
+  // Deserialize error table
+  std::string prefix =
+    this->simulation_parameters.simulation_control.output_folder;
+  std::string suffix = ".checkpoint";
+  if (this->simulation_parameters.analytical_solution->calculate_error())
+    deserialize_table(
+      this->error_table,
+      prefix + this->simulation_parameters.analytical_solution->get_filename() +
+        "_HT" + suffix);
+  if (this->simulation_parameters.post_processing.calculate_heat_flux)
+    {
+      deserialize_table(
+        this->heat_flux_table,
+        prefix +
+          this->simulation_parameters.post_processing.heat_flux_output_name +
+          suffix);
+    }
+  if (this->simulation_parameters.post_processing
+        .calculate_temperature_statistics)
+    {
+      deserialize_table(
+        this->statistics_table,
+        prefix +
+          this->simulation_parameters.post_processing.temperature_output_name +
+          suffix);
+    }
```

Comment:
Same question here

Reply:
I saw this a few times and it was always consistent, so not a problem unless you prefer to change it